### PR TITLE
Add OSD_TOTAL_PACKS (with betaflight/pull/10808)

### PR DESCRIPTION
**Pull request by bree29**
### Add OSD_TOTAL_PACKS

Satisfies `Needs coordination with betaflight-configurator` label on the [original PR #10808](https://github.com/betaflight/betaflight/pull/10808).

I tested the betaflight-configurator on Linux64 platform, and with modified Betaflight 4.3 (the one in PR) on STM32F7X2 target
```
# config: manufacturer_id: AIKO, board_name: AIKONF7, version: 34f72565, date: 2019-10-03T17:08:47Z
```

All settings went good on BF configurator and seems OK.

I'll test the feature itself this weekend ; all is looking good on the betaflight-configurator side.

### CODE REVIEW

File: src/js/tabs/osd.js
Comment (Asizon) [inline src/js/tabs/osd.js:1823]: It should be added to API_VERSION_1_44, the existing code seems wrong, @limonspb why you added Flight count in 1.42?

**Review by Asizon — CHANGES_REQUESTED** [review]

File: src/js/tabs/osd.js
Comment (Asizon) [inline src/js/tabs/osd.js:1725]: Why here is TOTAL_PACKS and not STAT_TOTAL_PACKS?

Comment (Asizon) [inline src/js/tabs/osd.js:1725]: Also this bit should go the last one on the list.

**Review by Asizon — CHANGES_REQUESTED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by bree29** [comment]
```
yarn
yarn test
yarn start
```

Feature tested on BF side (see original PR)

Reviews required by @Asizon have been made.

**Review by haslinghuis — DISMISSED** [review]

**Review by Asizon — APPROVED** [review]

File: locales/en/messages.json
Comment (limonspb) [inline locales/en/messages.json:5259]: flown packs could be better? Need a native English speaker here :)

**Comment by bree29** [comment]
Accidently closed the PR alongside my rebase. Everything should work great right now. Please let me now.

**Review by haslinghuis — DISMISSED** [review]

**Review by Asizon — APPROVED** [review]
LGTM!

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> FAIL
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by ctzsnooze — DISMISSED** [review]

**Comment by haslinghuis** [comment]
@bree29 please squash your commits. Attached `Don't merge` label awaiting merging of firmware part.

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> PASS
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> FAIL
 - `Don't merge` label NOT found -> FAIL
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by mikeller — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> PASS
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> FAIL
 - `Don't merge` label NOT found -> FAIL
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> PASS

**Comment by ctzsnooze** [comment]
@bree29 - If you want this merged into 10.8, please squash your commits and indicate that you still think it is important.

Otherwise we will tag it for 10.9 and think about it after 4.3 / 10.8 are released.

**Comment by bree29** [comment]
> @bree29 - If you want this merged into 10.8, please squash your commits and indicate that you still think it is important.
> 
> Otherwise we will tag it for 10.9 and think about it after 4.3 / 10.8 are released.

Just in case my updates from upstream will be an issue (the "one commit" thing, that were 5 here), just as it was in the betaflight part, I followed the same solution to update it to HEAD & squash my commit. Since I must do something wrong in my rebase sequences.

Sorry for the inconvenience of having to re-approving it for merging.

**Comment by haslinghuis** [comment]
Merging upstream changes does count as commit but is actually not when rebasing. Found out after using letting vscode to do this for me. So I'm back on the command line.

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> FAIL
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by Asizon** [comment]
Firmware pr is labeled for 4.4,i think we are not in time to merge this new feature into 4.3

**Review by limonspb — APPROVED** [review]

**Comment by bree29** [comment]
@limonspb , do I need to rebase before reviews are approved ? Same question for the firmware PR

**Review by haslinghuis — APPROVED** [review]

**Comment by haslinghuis** [comment]
@bree29 there are no conflicts. Both PR's are ready to merge. Thanks!

**Pull request closed without merging**
